corehq/apps/case_migrations/views.py

```diff
@@ -54,7 +54,7 @@ def get_case_and_descendants(domain, case_id):
             if not c.closed]
 
 
-@mobile_auth
+@mobile_auth_or_formplayer
```

Comment:
if i'm reading the auth functions correctly, for a given logged in user, this view would give access to any case in that user's domain if they know the case id. it seems like that's the way it was before too.

@wpride i think you mentioned that this case restore endpoint is no longer being used by anything else besides this new smsforms / formplayer integration, is that right?

if so then this should probably just be set to use formplayer auth with the mac digest and nothing else

Reply:
@gcapalbo Confirmed